### PR TITLE
Add drag and drop feature

### PR DESCRIPTION
**Pull request by kimothokamau**
This pull request aims to merge drag and drop feature to the development branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To developers, Microverse, ruby official guidelines and The odin Project
+
+## 📝 License
+
+This project is [MIT](https://es.wikipedia.org/wiki/Licencia_MIT) licensed.
```
Comment (MahmoudBakr23) [inline README.md]: - [x] Kindly follow this [guideline](https://github.com/microverseinc/readme-template) for how to generate a MIT license link. 😊

File: src/backend.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-mutable-exports */
+let todos = [
+  { description: 'read es6 arrow functions', completed: false, index: 1 },
+  { description: 'read on dom events', completed: false, index: 2 },
```
Comment (MahmoudBakr23) [inline src/backend.js:4]: - [ ] **OPTIONAL**
Please notice that you repeated this code once in the data file. Instead, you could use import and export keywords for that matter 😊
_Kindly keep in mind that you'll learn and be asked about this point in the upcoming projects.👌🏻_

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello team 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 
- Descriptive pr summary and headline 
- Live demo link
- Code separation 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by voscarmv — APPROVED** [review]
Hi @kimothokamau , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kimothokamau** [comment]
Thank you @voscarmv

**Pull request merged**